Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-5184/git-issue-5184.dfy

```diff
@@ -0,0 +1,9 @@
+// RUN: %exits-with 0 %verify "%s"
```

Comment:
You'll want to actually execute the program, since that's when the bug showed up:

```suggestion
// RUN: %testDafnyForEachCompiler "%s"
```

That will also make you define a `git-issue-5184.dfy.expect` file with the expected output.